recipe/meta.yaml

```diff
@@ -150,19 +157,80 @@ outputs:
         - if exist %PREFIX%\\Library\\lib\\parquet_static.lib exit 1        # [win]
         - if exist %PREFIX%\\Library\\lib\\gandiva_static.lib exit 1        # [win]
 
+  - name: pyarrow
+    script: build-pyarrow.sh  # [not win]
+    script: bld-pyarrow.bat   # [win]
+    version: {{ version }}
+    build:
+      number: {{ number }}
+      string: "{{ build_string }}"
+      skip: true  # [cuda_compiler_version not in (undefined, "None", "9.2")]
+      track_features:
+        {{ "- arrow-cuda" if cuda_enabled else "" }}
+    requirements:
+      build:
+        - cmake 3.16.*
+        - ninja
+        - make  # [unix]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        # pyarrow does not require nvcc but it needs to link against libraries in arrow-cpp=*=*cuda
+        - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
+      host:
+        # directly pin boost-cpp as we also seem to directly include boost symbols
+        # in the Python modules.
+        - arrow-cpp ={{ version }}=*{{ build_ext }}
+        - boost-cpp
```

Comment:
The comment for boost and the line where it's on don't fit together anymore.

Reply:
@pearu, in contrast to the pyarrow-feedstock, I'm not a collaborator here, so I cannot push to this PR's branch. In any case, there's only details left from my POV.

Reply:
Thanks, @h-vetinari, I'll fix the nit and wait for an extra 24h to give anyone time for review.

Reply:
Fixed.